src/pages/index.js

```diff
@@ -122,7 +123,6 @@ const newPostForm = new ModalWithForm({
           link: inputValues[cardSelectors.cardInputLink],
         })
         .then((res) => {
-          formValidators[newPostForm.formName].resetValidation();
           cardSection.createAndRenderItem(res);
         });
     }
```

Comment:
And `onDeleteAPIManager` should be changed a bit. You should `return` the promise from the **api** call like this


```js


  const onDeleteAPIManager = (deleteConfirmationManager, id) => {
    function makeRequest() {
      return deleteConfirmationManager.awaitUserChoice().then((choice) => {
        if (choice) {
// here you return the promise to be able to see `Deletin...` in the button
          return dbApi.requestDelete(id).then(() => {
            newCard.remove();
          });
        }
      });
    }

```